components/user/ChangeEmail.vue

```diff
@@ -7,21 +7,28 @@
         class="flex flex-col p-6 pt-0 space-y-2"
         @submit.prevent="addNewEmail"
       >
-        Enter the new email you wish to switch to, and your password.
         <text-input
           v-model="email"
           name="New Email"
```

Comment:
So fun fact, I changed this component fairly significantly a while back, where I removed a label that was bundled inside the `text-input` component. I have clearly missed a lot of cases where we used this functionality!
We can remove the `name` attribute from this (and password below), as they were only used to automatically display a level for the text input. 

If you wanted to re-instate the label (vs a placeholder, which I am absolutely fine with btw!), you just need to wrap inside a `UiFormLabel` component (e.g. https://github.com/BristolSTA/uobtheatre-web/blob/develop/components/auth/AuthBox.vue#L50)

Reply:
I'm pretty torn! Placeholders have a certain elegance:

![image](https://user-images.githubusercontent.com/25545513/230734971-911c9afc-f741-4157-bb98-09bdbc060308.png)

Whereas labels maybe feel a bit more divided up?

![image](https://user-images.githubusercontent.com/25545513/230735592-98b30906-448d-4285-8253-d872c579f33c.png)

I'm honestly leaning towards placeholders myself, but if you think strongly otherwise that's fine too haha

Reply:
i like the placeholders more